packages/create-sitecore-jss/src/templates/nextjs-styleguide/src/lib/next-config/plugins/disconnected.js

```diff
@@ -13,8 +13,10 @@ const disconnectedPlugin = (nextConfig = {}) => {
 
   return Object.assign({}, nextConfig, {
     async rewrites() {
+      const existingRewrites = await (nextConfig.rewrites ? nextConfig.rewrites() : []);
```

Comment:
Good catch combining with existing rewrites. It was fine to override these outright at one point, but we've got new ones (and possible a customer adds them) which should remain.

Reply:
Right, potentially it can happen